service_capacity_modeling/capacity_planner.py

```diff
@@ -577,7 +577,10 @@ def _plan_certain(  # pylint: disable=too-many-positional-arguments
         plans.sort(key=lambda p: (p.rank, p.candidate_clusters.total_annual_cost))
 
         num_results = num_results or self._default_num_results
-        return reduce_by_family(plans)[:num_results]
+        max_per_family = extra_model_arguments.get("max_results_per_family", 1)
```

Comment:
If we do end up lifting to planner, we'd probably codify in the api as a kwarg, since it's not a parameter of the workload (the desires) but rather the plan. So like how num_results or lifecycles/instance_familes etc work above